mods/cnc/rules/defaults.yaml

```diff
@@ -480,10 +480,13 @@
 		SellSounds: cashturn.aud
 	Capturable:
 	WithMakeAnimation:
+		MakeUpgrades: underConstruction
+	DisableOnUpgrade:
+		UpgradeTypes: underConstruction
```

Comment:
Yes, that has been our whole argument.
Adding more and more inconsistency is bad, no matter how slowly.